### PR TITLE
Some miscellaneous fixes

### LINKED ISSUE #331: [BUG] Being Cloned Causes Removable Colorblindness
Copy of https://github.com/Whitesands13/Whitesands/issues/561

## Reproduction:
1) Build a complete cloning setup
2) Get scanned
3) Die
4) Get cloned (or drag your aghost into an empty clone)
5) Be colorblind

Workaround:
Equipping and removing a welding helmet removes this colorblindness

### LINKED ISSUE #388: [BUG] Syndicate Sleepers cannot be controlled from inside
The Syndicate Sleeper has a "controls_inside" variable that's supposed to give it the ability to be used from inside, for self-healing purposes.
The code that controls how this var functions is here, under sleeper.dm:
code
But it seems to be completely nonfunctional at the moment, likely related to whitesand's sleeper rework.

## Round ID:
n/a

## Reproduction:
Locate syndicate sleeper. Climb inside and watch the control ui disappear. Cry.

### PR DESCRIPTION
**Pull request by Bokkiewokkie**
## About The Pull Request

Renames the .dme to shiptest,
Fixes #388,
Fixes #331,
Makes some areas have proper names,
I haven't tested any of this but it compiled so that's good enough.

## Why It's Good For The Game

image

## Changelog
:cl:
fix: Hopefully made cloning no longer make monochromacy temporary.
fix: Shiptest maintenance areas are now no longer named "Hallway".
fix: Allows you to access Syndicate sleepers from the inside again.
code: The dream maker enviroment has been renamed to shiptest.
/:cl:

### CODE REVIEW

**Comment by AtlasRules** [comment]
damn he wasn't lying todays mothpost really was moved to github PR #389

File: whitesands/code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -213,12 +213,9 @@
 		visible_message("<span class='notice'>[usr] pries open [src].</span>", "<span class='notice'>You pry open [src].</span>")
 		open_machine()
 
-/obj/machinery/sleeper/ui_state(mob/user)
-	if(controls_inside)
-		return GLOB.notcontained_state
-	return GLOB.default_state
-
 /obj/machinery/sleeper/ui_interact(mob/user, datum/tgui/ui)
+	if(src.contains(user) && !controls_inside)
```
Comment (ZephyrTFA) [inline whitesands/code/game/machinery/Sleeper.dm:217]: You don't need src here. And also override UI_STATUS to return UI_CLOSE if inside and no controls inside aswell

Comment (Bokkiewokkie) [reply whitesands/code/game/machinery/Sleeper.dm:217]: But why shouldn’t they be able to watch which buttons someone presses on the sleeper if they were already looking at it :)?

Comment (ZephyrTFA) [reply whitesands/code/game/machinery/Sleeper.dm:217]: Can't see the panel if you're inside

Comment (Bokkiewokkie) [reply whitesands/code/game/machinery/Sleeper.dm:217]: Yeah I’ll do it tomorrow

**Pull request merged**